backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/util/TimeUtil.java

```diff
@@ -2391,4 +2403,21 @@ public static long getDaysBetween360(@NonNull final Instant from, @NonNull final
 		return getDaysBetween360(asZonedDateTime(from), asZonedDateTime(to));
 	}
 
+	@NonNull
+	public static Timestamp roundDownToNearestQuarter(@NonNull final Timestamp initialDate)
```

Comment:
why not using org.compiere.util.TimeUtil#trunc and just support a new TRUNC_15m constant?